crates/node/src/lib.rs

```diff
@@ -385,7 +385,7 @@ async fn handle_line(node: &mut Node, line: String) -> eyre::Result<()> {
                                 Some((context, application, version, path, hash))
                             })
                         else {
-                            println!("{IND} Usage: context create <context_id> <application_id> <version> <path> <hash>");
+                            println!("{IND} Usage: context create <context_id> <application_id> <version> <path>");
```

Comment:
I meant we don't need hash in the CLI at all, so it doesn't need to be parsed in 385 either

Reply:
Updated